onnx_chainer/export.py

```diff
@@ -137,17 +137,19 @@ def create_node(
     def backward_postprocess(self, function, in_data, out_grad):
         if isinstance(function, chainer.function.FunctionAdapter):
             function = function.function
-        func_name = function.__class__.__name__
+        func_name = getattr(function, 'user_name', function.__class__.__name__)
         temp_node_name = '{}:{}'.format(
             self.func_name_counts[func_name], func_name)
         self.func_name_counts[func_name] += 1
 
         input_names = []
-        for i in function.inputs:
-            # 'i' is a VariableNode, so check if it has a Variable/Parameter
-            var = i.get_variable_or_none()
+        for input_ref in function.inputs:
```

Comment:
s/input_ref/input/ ? I think this isn't a weakref?

Reply:
Exactly, thx